src/cata_tiles.cpp

```diff
@@ -2124,6 +2124,13 @@ bool cata_tiles::draw_sprite_at(
                 break;
             case 1:
                 // 90 degrees (and 270, with just two sprites)
+#if defined(_WIN32) && defined(CROSS_LINUX)
+                // For an unknown reason, additional offset is required in direcr3d mode
+                // for cross-compiling from Linux to Windows
+                if( get_option<std::string>( "RENDERER" ) == "direct3d" ) {
```

Comment:
Accessing options and string compare is pretty expensive. Especially when it would be done for each rendered tile. It would be better to make it a boolean option (maybe guarded by cross compile macro) and cache this value to a variable.

Reply:
Ok, I will think about it next day.

Reply:
Done.